ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -235,12 +235,12 @@ function run_command() {
 
 # Check if a build is signed
 function check_signed() {
-    local digest algorithm hash_value response try max_retries
-    if [[ "${TARGET}" =~ "@sha256:" ]]; then
-        digest="$(echo "${TARGET}" | cut -f2 -d@)"
+    local digest algorithm hash_value response try max_retries payload="${1}"
```

Comment:
For better maintenance, shall we keep the common func check_signed consistent with other steps?